aws/resource_aws_inspector_resource_group_test.go

```diff
@@ -17,40 +23,67 @@ func TestAccAWSInspectorResourceGroup_basic(t *testing.T) {
 			{
 				Config: testAccAWSInspectorResourceGroup,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAWSInspectorResourceGroupExists("aws_inspector_resource_group.foo"),
+					testAccCheckAWSInspectorResourceGroupExists(resourceName, &v1),
+					testAccMatchResourceAttrRegionalARN(resourceName, "arn", "inspector", regexp.MustCompile(`resourcegroup/.+`)),
+					resource.TestCheckResourceAttr(resourceName, "tags.Name", "foo"),
 				),
 			},
 			{
 				Config: testAccCheckAWSInspectorResourceGroupModified,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAWSInspectorResourceGroupExists("aws_inspector_resource_group.foo"),
+					testAccCheckAWSInspectorResourceGroupExists(resourceName, &v2),
+					testAccMatchResourceAttrRegionalARN(resourceName, "arn", "inspector", regexp.MustCompile(`resourcegroup/.+`)),
+					resource.TestCheckResourceAttr(resourceName, "tags.Name", "bar"),
+					func(s *terraform.State) error {
```

Comment:
Nit: We typically create these as separate `TestCheckFunc` so they are reusable across multiple tests if desired, see also other `testAccCheck...Recreated` functions 👍

```
func testAccCheckAWSInspectorResourceGroupRecreated(i, j *inspector.ResourceGroup) resource.TestCheckFunc {
	return func(s *terraform.State) error {
		if aws.TimeValue(i.CreatedAt) == aws.TimeValue(j.CreatedAt) {
			return fmt.Errorf("Inspector Resource Group not recreated")
		}
		return nil
	}
}
```

Reply:
Done.